source/connection-monitoring-and-pooling/tests/README.rst

```diff
@@ -36,7 +36,22 @@ Unit Test Format:
 
 All Unit Tests have some of the following fields:
 
-- ``poolOptions``: if present, connection pool options to use when creating a pool
+- ``poolOptions``: If present, connection pool options to use when creating a pool;
+  both `standard ConnectionPoolOptions <https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#connection-pool-options-1>`__
+  and the following test-specific options are allowed:
+
+  - ``backgroundThreadIntervalMS``: A desired time interval between the end of a
```

Comment:
@patrickfreed  should we also mention `backgroundThreadIntervalMS` as internal option or interval between Runs in general in [background thread definition](https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst#background-thread), as it has direct impact on background thread implementation?

Reply:
I preemptively (while there is no response) fleshed out a bit possible restrictions a test runner may introduce.